flow/order/update_shipping_address.yaml

```diff
@@ -0,0 +1,50 @@
+steps:
+  - say: Right! I'll need your new address!
+
+  - ask_form: What's the new shipping address?
+  - flow_set: address
+
+  - ask_form: Address 2?
+  - flow_set: address_2
+
+  - ask_form: City?
+  - flow_set: city
+
+  - ask_form: Postal code?
+  - flow_set: postal_code
+
+  - ask_form: Country?
+  - flow_set: country
+
+  - ask_form: Province?
+  - flow_set: province
+
+  - type: meya.shopify.component.order.update_shipping_address
+    order_id: (@ flow.order_id )
+    address: (@ flow.address )
+    address_2: (@ flow.address_2 )
+    city: (@ flow.city )
+    postal_code: (@ flow.postal_code )
+    country: (@ flow.country )
+    province: (@ flow.province )
+    integration: integration.shopify
+
+  - if: (@ flow.result)
```

Comment:
How does this check for an error? Isn't `flow.result` just an order?

Reply:
the api request expects for a 200 or 201, if for whatever reason that's not the response, flow.result would be None, right?
I didn't test this specifically to be honest.

Reply:
In this case `flow.result` won't get set, the flow will stop due to the exception. You can remove the in-flow error handling then.